app/views/records/edit_fields/_rights.html.erb

```diff
@@ -7,7 +7,7 @@
         end ) + 
         ( content_tag :aside do
             t_uri(:description, scope: [ :rights, b.value ])
-        end )
+        end ) if !b.text.include?('3.0')
```

Comment:
@grosscol: Cannot find a simple way to do it. I thought this is a pretty clean and straightforward solution. I will need to update the tests though!

 If you have some ideas, do let me know.